packages/fileutils/fileutils.0.6.1/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+maintainer: "Sylvain Le Gall <sylvain+ocaml@le-gall.net>"
+authors: [ "Sylvain Le Gall" ]
+homepage: "https://github.com/gildor478/ocaml-fileutils"
+license: "LGPL-2.1 with OCaml linking exception"
+dev-repo: "git+https://github.com/gildor478/ocaml-fileutils.git"
+bug-reports: "https://github.com/gildor478/ocaml-fileutils/issues"
+doc: "https://gildor478.github.io/ocaml-fileutils/"
+
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name] {with-test}
+  ["dune" "build" "@doc" "-p" name] {with-doc}
+]
+depends: [
+  "ocaml"
+  "base-unix"
+  "base-bytes"
+  "stdlib-shims"
+  "dune" {build & >= "1.11.0"}
```

Comment:
@gildor478 dune shouldn't be marked as `build`, sorry I didn't know it when I switched the project to dune. See #14266.

Reply:
I have fixed both this PR and the upstream opam files.